.editorconfig

```diff
@@ -9,6 +9,10 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.js]
```

Comment:
True, that could have been a separate PR.

All JavaScript files are right now using exactly this configuration. When you look into the code, they use tabs. My editor was picking up on the wrong config and was doing bad stuff...